storage/google_drive/drive_client_wrapper.py

```diff
@@ -0,0 +1,135 @@
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+from googleapiclient.http import MediaFileUpload
+
+import mimetypes
+import os
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = 'https://www.googleapis.com/auth/drive'
+
+drive_service = None
+drive_folder_root = None
+
+def init_client(auth_credentials_path):
```

Comment:
I don't think this path is being used?

It might be worth specifying the two paths (credentials and token) separately to make it clear that you need both - as well as a readme pointing out how to generate them

Reply:
Thanks for catching this, yes, auth_credentials_path was supposed to replace "credentials.json".

I've added the two paths separately as you suggested, and added more comments here as to what happens in the oauth flow. As far as I can tell, one needs a browser to generate the auth tokens, so for now I've left the code to do that here. As we discussed offline, we may want to create a separate tool for doing this.